source/app/web/statics/style.css

```diff
@@ -1,361 +1,380 @@
 /* General */
-  body {
-    font-family: -apple-system,BlinkMacSystemFont,Segoe UI,Helvetica,Arial,sans-serif,Apple Color Emoji,Segoe UI Emoji;
-    margin: 0;
-    padding: 0;
-    background-color: var(--color-bg-canvas);
-    color: var(--color-text-primary);
-  }
-
-  iframe {
```

Comment:
You should undo the rest of the changes and only include your edits.

Reply:
That's happened due to the code formatter extension, there was no code change but only relevant code lines were added.
Though on your request I made new pull request #903.